src/Components/ReviewScore/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { ReviewStandard } from '@/Types/review';
+import styled from 'styled-components';
+import { StarScore } from './StarScore';
+
+interface ReviewScoreProps {
+  /** 리뷰 평가 기준 */
+  standard: ReviewStandard;
+  /** 평가 점수 */
+  score: number;
+}
+
+export const ReviewScore = ({ standard, score }: ReviewScoreProps) => {
+  return (
+    <ReviewScoreBox>
+      <ReviewScoreRow>
+        <StarScore score={score} />
+        <ReviewScoreText $score={score}>{score}</ReviewScoreText>
+      </ReviewScoreRow>
+      <ReviewStandardText>{standard}</ReviewStandardText>
+    </ReviewScoreBox>
+  );
+};
+
+const ReviewScoreBox = styled.li`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ReviewScoreRow = styled.div`
+  position: relative;
+  width: 48px;
+  height: 48px;
+`;
+
+const ReviewScoreText = styled.p<{ $score: number }>`
+  position: absolute;
+  top: 0;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -40%);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: ${({ theme, $score }) => ($score <= 2 ? theme.color.black5 : theme.color.white)};
```

Comment:
score에 따라 텍스트 색상이 달라지는 부분을 잘 구현해주신 것 같아요 👍
다만, 피그마 상에선 2가 채워진 부분은 흰색, 안채워진 부분은 검정색으로 보이고 있어서 아래처럼 작성하는 것을 제안드립니다.

기존: 2가 검정색 단일 색상으로 보임
<img width="379" alt="image" src="https://github.com/Ludo-SMP/ludo-frontend/assets/71035113/5e894d7a-c09b-442a-bc29-0aee22e01db8">

제안: 2의 경우 하단 40%는 흰색, 상단 60%는 검정색으로 보임
<img width="372" alt="image" src="https://github.com/Ludo-SMP/ludo-frontend/assets/71035113/b4195a4a-de97-48b3-831a-d38c48995027">

참고: 피그마 star 컴포넌트 
https://www.figma.com/design/OLgSF11mOFN5MpcYzpD3N3/Design-System?node-id=3142-1749&m=dev


```suggestion
  ${({ theme, $score }) =>
    $score === 2
      ? css`
          background-clip: text;
          background-image: linear-gradient(
            to top,
            ${theme.color.white} ${$score * 20}%,
            ${theme.color.black5} ${$score * 20}%
          );
          color: transparent;
          -webkit-text-fill-color: transparent;
        `
      : css`
          color: ${$score <= 2 ? theme.color.black5 : theme.color.white};
        `}
```

Reply:
styled-component의 css를 활용해서, linear-gradient 방식으로 text 색상을 부분마다 다르게 부여할 수 있군요. 좋은 의견감사합니다!!. 반영하겠습니다.!! 🙏